programs/bpf_loader/src/syscalls/mod.rs

```diff
@@ -319,12 +319,12 @@ pub(crate) fn morph_into_deployment_environment_v1(
     // config.enabled_sbpf_versions =
     //     *config.enabled_sbpf_versions.end()..=*config.enabled_sbpf_versions.end();
 
-    let mut result = BuiltinProgram::new_loader_with_dense_registration(config);
+    let mut result = BuiltinProgram::new_loader(config);
 
-    for (key, (name, value)) in from.get_function_registry(SBPFVersion::V3).iter() {
+    for (_key, (name, value)) in from.get_function_registry().iter() {
```

Comment:
What about using `.values`?

Reply:
Hehe, I fell into the same trap. No, we haven't implemented that.